rpc/connector/connector.proto

```diff
@@ -161,7 +164,7 @@ enum InterceptError {
   NO_ACCEPTABLE_WORKLOAD = 8;
   AMBIGUOUS_MATCH = 9;
   FAILED_TO_ESTABLISH = 10;
-  reserved 11;
```

Comment:
Aren't we supposed to not use numbers that were reserved in the past? or is it not a big deal since these are all InterceptErrors?

Reply:
We are supposed to do that where backward compatibility can be an issue. That's not the case with the connector protocol because telepresence of later versions will refuse to continue if there's a version mismatch between the cli and the connector.